tola/tests/test_util.py

```diff
@@ -245,3 +249,131 @@ def test_save_data_to_silo_skipped_rows(self):
 
         result = save_data_to_silo(self.silo, data, self.read)
         self.assertEqual(result, expected_response)
+
+    @patch('logging.warning')
+    def test_ona_parse_type_group_with_valid_data(self, mock_logger):
```

Comment:
It's good to mock the function `ona_parse_type_group`. It's called in the function. `ona_parse_type_group`